readme.md

```diff
@@ -80,6 +80,11 @@ This plugin leverages the following third-party libraries for various functional
 
 ## Installation
 
+### From the WordPress.org plugin repository
+1. Install it in your WordPress instance: go to WP Admin Plugins > Add new plugin. Search for `gatherpress`. Once found, click `Install`.
+2. Activate the plugin.
+
+### From the file downloaded from our GitHub repository
 1. Download the plugin: you'll find the latest release on the GatherPress GitHub repository, under [Releases](https://github.com/GatherPress/gatherpress/releases) > Assets and download `gatherpress.zip`.
```

Comment:
Linking to https://github.com/GatherPress/gatherpress/releases/latest will redirect to the latest tagged release.

Reply:
… sorry, taping the correct lines on my tablet wasn’t as intuitive as I thought. But you’ll get it.